README.md

```diff
@@ -1,3 +1,62 @@
-# juno-sdk
+# Juno SDK
 
-Juno dev SDK
+**Juno SDK** is a development toolkit for interacting with Juno's central infrastructure. It provides convenient models and API methods for handling common tasks like authentication, email management, user management, and project-related operations. This document outlines the core structure of the SDK.
+
+## SDK Structure
+
+### Models
+
+The SDK uses several models to structure the data exchanged between the client and Juno's services. These models represent the input data for API requests and the output data returned by the server.
+
+#### Key Models:
+- **CreateProjectModel**: Defines the structure for creating a new project.
```

Comment:
The problem with having key models here is in the event we change/add/remove, we then need to manually update in the README. The chances of someone actually remembering to do this is rather low, so we'll likely need to have a link to the online documentation. However, considering the open docs are attached to a currently running instance we'll need to decide whether we want the link here or not (e.g. susceptible to unwanted visitors).